lib/xlsx_reader/parsers/worksheet_parser.ex

```diff
@@ -103,8 +105,17 @@ defmodule XlsxReader.Parsers.WorksheetParser do
     {:ok, expect_value(state)}
   end
 
-  def handle_event(:start_element, {"f", _attributes}, state) do
-    {:ok, expect_formula(state)}
+  def handle_event(:start_element, {"f", attributes}, state) do
+    attributes = attributes |> Map.new()
```

Comment:
We could avoid converting the whole structure into a map just to lookup the shared formula index. 
We could use the `Utils.get_attribute/3` to pluck the `"si"` attribute and handle this case.

Reply:
Done.